src/internal/eval-tidy.c

```diff
@@ -152,9 +152,30 @@ static bool is_data_mask(sexp* env) {
     r_typeof(env) == r_type_environment &&
     r_env_has(env, data_mask_flag_sym);
 }
-
+sexp* rlang_is_data_mask(sexp* env) {
+  return r_lgl(is_data_mask(env));
+}
 static sexp* data_pronoun_sym = NULL;
 
+static bool data_mask_has(sexp* mask, sexp* sym) {
+  sexp* top_env = r_env_get(mask, data_mask_top_env_sym);
```

Comment:
I think it should be `r_env_find()` because the `get` variant evaluates. This will cause a rchk note about protection. I'm planning to remove `r_env_get()` from the API for this reason.

Reply:
ok but we only want to search for it in `mask`, not further